2022/31xxx/CVE-2022-30190.json

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
Expected location (2022/30xxx/CVE-2022-30190.json) of file does not match actual location (2022/31xxx/CVE-2022-30190.json)